docs/chapters/workflow/release/howto-release-pelux.rst

```diff
@@ -0,0 +1,98 @@
+Creating a PELUX release
+========================
+
+Tasks needed to prepare PELUX for release
+-----------------------------------------
+#. Choose version number.
+#. Decide what Yocto release to follow.
+#. Decide what Qt version to support.
+#. Prepare the manifest and layers.
+#. Prepare the Software Factory Baseline.
+#. Prepare the release notes.
+#. Announce the release.
+#. Post-release retrospective.
+
+How-to perform the tasks
+------------------------
+In many of the tasks below, one is required to create both a branch and a tag.
+The rationale for this is that minor changes made after a release should go into
+the same branch, but will get a new tag with an updated minor version number.
+
+Before touching any files
+^^^^^^^^^^^^^^^^^^^^^^^^^
+* Choose version number. If this is a regular release, it should be ``X.0``
+  where X is the major version from the latest release incremented by one.
+  If this is a minor release (bugfixes, typos etc), it will be ``X.Y``, where X
+  is the major version from the latest release, and Y is the minor version
+  number from latest release incremented by one.
+* Decide on what Yocto release to follow. If this is a major release then
+  this is usually the latest Yocto release. If this is minor release, then
+  this was already decided when the major release was done, and you should
+  not change it.
+* Decide on what Qt version to support. The Qt project does not follow the
+  same release cadence as the Yocto project does. This will typically be the
+  latest Qt version on a major release, and the same as for the major
+  release in a minor release.
+
+Prepare the manifest
```

Comment:
Shouldn't this be "Prepare the PELUX baseline manifest" to disambiguate between this manifest and the Yocto manifest files?

Reply:
Well, we only have one manifest, and we are actually talking about the Yocto manifest files here. But perhaps it would make sense to reiterate that it is the baseline we're talking about.

Reply:
Thinking out loud I'd say that "the manifest" is pretty correct since we should have one manifest for the release and the fact that it's for a release of the PELUX Baseline project is given by the context. This is documentation for that project only.

In Blueprint-land I could imagine texts mentioning that there are "manifest files used for this and that and projects need  to define theirs like so and so", but that should not be a relevant abstraction level in these sections here.